torch/onnx/_internal/fx/_pass.py

```diff
@@ -161,8 +168,33 @@ def __init__(
             diagnostic_context: The diagnostic context for recording diagnostics.
             module: The module to be transformed.
         """
-        self.module = module
         self.diagnostic_context = diagnostic_context
+        self.module = module
+        self.fake_mode = self._detect_fake_mode()
+
+    def _detect_fake_mode(self) -> Optional[fake_tensor.FakeTensorMode]:
+        """Detect fake mode from the graph.
+
+        Scan through all nodes in graph and their meta['val'] to detect fake mode.
+        """
+        fake_tensors = []
+        for node in self.module.graph.nodes:
+            try:
+                fake_tensors.append(node.meta.get("val"))
+            except RuntimeError:
+                continue
```

Comment:
Comment on when the error is expected?

Reply:
good catch, I think it was left over when the internal code was calling another function. I'll remove